litr/src/main/java/com/linkedin/android/litr/transcoder/VideoTrackTranscoder.java

```diff
@@ -40,8 +42,9 @@ public class VideoTrackTranscoder extends TrackTranscoder {
 
     @NonNull private MediaFormat sourceVideoFormat;
     @NonNull private MediaFormat targetVideoFormat;
+    @Nullable private FrameDropper frameDropper;
 
-    VideoTrackTranscoder(@NonNull MediaSource mediaSource,
+    public VideoTrackTranscoder(@NonNull MediaSource mediaSource,
```

Comment:
Does this have to become public?

Reply:
Nope, apologies, I added my test to the wrong package so it wasn't visible. Updated at: [VideoTrackTranscoder.java#L48](https://github.com/linkedin/LiTr/blob/a6d524823017dc2cf30ed85026e7d1ad3af4ace9/litr/src/main/java/com/linkedin/android/litr/transcoder/VideoTrackTranscoder.java#L48) and I've moved the frame dropper tests to the correct package.

Reply:
Nope, apologies, I added my test to the wrong package so it wasn't visible. Updated at: [VideoTrackTranscoder.java#L48](https://github.com/linkedin/LiTr/blob/a6d524823017dc2cf30ed85026e7d1ad3af4ace9/litr/src/main/java/com/linkedin/android/litr/transcoder/VideoTrackTranscoder.java#L48) and I've moved the frame dropper tests to the correct package.